src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateDirectionQuestionnaireLipCallBackHandler.java

```diff
@@ -59,18 +63,34 @@ private CallbackResponse prepareDirectionsQuestionnaire(CallbackParams callbackP
         CaseDocument sealedDQForm = directionQuestionnaireLipGeneratorFactory
             .getDirectionQuestionnaire()
             .generate(caseData, callbackParams.getParams().get(BEARER_TOKEN).toString());
-        caseDataBuilder
-            .systemGeneratedCaseDocuments(systemGeneratedDocumentService
-                                              .getSystemGeneratedDocumentsWithAddedDocument(sealedDQForm, caseData));
-        if (sealedDQForm.getDocumentName().contains("defendant")) {
-            assignCategoryId.assignCategoryIdToCaseDocument(sealedDQForm, DocCategory.DQ_DEF1.getValue());
-        } else {
+        if (featureToggleService.isGaForWelshEnabled()
+            && sealedDQForm.getDocumentName().contains("claimant")
+            && (caseData.isClaimantBilingual() || caseData.isRespondentResponseBilingual())) {
```

Comment:
I have modified this condition in CIV-16891 - plz add that 
 || caseData.isLipClaimantSpecifiedBilingualDocuments()
            || caseData.isLipDefendantSpecifiedBilingualDocuments())